middlewares/optgeneretor.js

```diff
@@ -0,0 +1,37 @@
+
+import nodemailer from "nodemailer"
+
+
+//*using simple functions
+function generator() {
+    const randomNumber = Math.floor(Math.random() * 1000000);
+    //*by using the padStart we're making sure that it'll have the 6 digits code 
+    const otp = randomNumber.toString().padStart(6, '0');
+    const transporter = nodemailer.createTransport({
+    host: 'smtp.ethereal.email',
+    port: 587,
+    auth: {
+        user: 'emerson.ebert@ethereal.email',
+        pass: 'QNFN4vFS1Eg7vgMJCj'
+    }
+  });
+
+  const mailOptions = {
+    from: 'sameer.ijaz100@gmail.com',
+    to: 'emerson.ebert@ethereal.email',

```

Comment:
why it is hard coded? Get it from user request.

Reply:
okay sir